core/arch/arm/mm/tee_mm.c

```diff
@@ -35,19 +35,19 @@ static void pfree(tee_mm_pool_t *pool, void *ptr)
 		free(ptr);
 }
 
-bool tee_mm_init(tee_mm_pool_t *pool, paddr_t lo, paddr_t hi, uint8_t shift,
+bool tee_mm_init(tee_mm_pool_t *pool, paddr_t lo, size_t size, uint8_t shift,
 		 uint32_t flags)
 {
 	if (pool == NULL)
 		return false;
 
 	lo = ROUNDUP(lo, 1 << shift);
-	hi = ROUNDDOWN(hi, 1 << shift);
+	size = ROUNDDOWN(size, 1 << shift);
```

Comment:
We must decrease `size` with the amount of bytes `lo` was rounded up with first.

Reply:
ok, I'll update